cookiecutter/{{cookiecutter.github_repo_name}}/.github/workflows/ci.yaml

```diff
@@ -302,14 +302,18 @@ jobs:
     if: github.event_name != 'push'
     runs-on: ubuntu-20.04
     steps:
+      - name: Setup Git
+        uses: frequenz-floss/gh-action-setup-git@v0.x.x
+        # TODO(cookiecutter): Uncomment this for projects with private dependencies
```

Comment:
Any disadvantage if we just always set this?

Reply:
I just am very paranoid about passing around credentials where they are not really needed. :shrug: